biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -563,6 +607,38 @@ public void annotation(Annotation a) {
 								// Ignore
 							}
 						}
+						break;
+					case "org.osgi.annotation.bundle.Export" :
```

Comment:
I think we also need support for the `substitution` annotation element.

Reply:
That does need to be done, but I have no idea how best to implement it. There isn't an obvious equivalent from the bnd annotations, and I'm not sufficiently familiar with how bnd determines version range policies to feel comfortable making up a solution

Reply:
Then we need to open an issue to make sure we do not forget to implement this support at some point.

Reply:
I have commented in #1343 to say which items are outstanding (there are a few).